_modules/hubble.py

```diff
@@ -404,7 +404,7 @@ def sync():
     '''
     log.debug('syncing nova modules')
     nova_dir = __salt__['config.get']('hubblestack.nova.dir', 'salt://hubblestack_nova')
-    saltenv = __salt__['config.get']('hubblestack.nova.saltenv', 'base')
+    saltenv = saltenv or __salt__['config.get']('hubblestack.nova.saltenv', 'base')
```

Comment:
Won't this always choose `saltenv`? Because `'base'` is truthy. Probably need to set the default value to `None`.